eZ/Publish/Core/REST/Common/Values/Root.php

```diff
@@ -17,4 +17,21 @@
  */
 class Root extends RestValue
 {
+    /**
+     * @var Resource[]
+     */
+    protected $resources;
+
+    public function __construct($resources)
```

Comment:
You should make the argument option. Limits BC breaks. You should also typehint as an array.